File: src/main/java/com/procurement/submission/config/ApplicationConfig.java
package com.procurement.submission.config;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.procurement.submission.utils.DateUtil;
import com.procurement.submission.utils.JsonUtil;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

@Configuration
@Import({
        WebConfig.class,
        ServiceConfig.class,
        DatabaseConfig.class,
        ConverterConfig.class
})
public class ApplicationConfig {
    @Bean
    public JsonUtil jsonUtil(final ObjectMapper mapper) {
        return new JsonUtil(mapper);
    }

    @Bean
    public DateUtil dateUtil() {
        return new DateUtil();
    }
}

File: src/main/kotlin/com/procurement/submission/infrastructure/configuration/ApplicationConfig.kt
package com.procurement.submission.infrastructure.configuration

import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import

@Configuration
@Import(
    DaoConfiguration::class,
    ServiceConfig::class,
    WebConfig::class,
    ObjectMapperConfig::class,
    TransformConfiguration::class,
    LoggerConfiguration::class
)
class ApplicationConfig
